projects/demo/src/modules/components/icon/examples/9/index.less

```diff
@@ -0,0 +1,42 @@
+:host {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 0.625rem;
+
+    --x1-size: 1rem;
```

Comment:
I think variables overcomplicate example, let's skip them. Maybe we can even inline styles like [style.font-size.rem]="1" so that it's all in one line and in one place in HTML all available at a glance?

Reply:
fixed